src/ecalc/libraries/libecalc/common/libecalc/core/models/compressor/sampled/compressor_model_sampled.py

```diff
@@ -226,7 +226,6 @@ def evaluate_rate_ps_pd(
                     power_unit=Unit.MEGA_WATT,
                     inlet_stream_condition=CompressorStreamCondition(
                         pressure=list(suction_pressure) if suction_pressure is not None else None,
-                        actual_rate_m3_per_hr=list(rate) if rate is not None else None,
```

Comment:
Add a comment here or in `CompressorStreamCondition` to specify what you mentioned in the PR comment that knowing `fluid_density` is a requirement to set this, to avoid doing the same mistake in the future, and to bring awareness?